enterprise/api/v1/serializers.py

```diff
@@ -391,3 +391,16 @@ def to_representation(self, instance):
             for course_run in course['course_runs']:
                 course_run['enrollment_url'] = enterprise_customer.get_course_run_enrollment_url(course_run['key'])
         return updated_program
+
+
+class EnterpriseCustomerReportingConfigurationSerializer(serializers.ModelSerializer):
+    """
+    Serializer for EnterpriseCustomerReportingConfiguration model.
+    """
+
+    class Meta:
+        model = models.EnterpriseCustomerReportingConfiguration
+        fields = (
+            'enterprise_customer', 'active', 'delivery_method', 'email', 'frequency', 'day_of_month', 'day_of_week',
+            'hour_of_day', 'password'
```

Comment:
so this password thing (as previously discussed) feels wrong. I'm thinking maybe there is a shared secret we can configure between enterprise/edxapp and jenkins so that this can be encrypted and then decrypted for use when the script runs?

Reply:
How will the Enterprise administrator get access to the password so that they can share it with the Enterprise customer receiving the data?

Reply:
i guess we could add a property to the django admin that would decrypt the password for viewing if someone goes into the detail view?

Reply:
Oh nevermind. You are talking specifically about making the password available in the data returned from the API endpoint.

Reply:
Yeah, you could store the shared secret in [edx-ops/business-secure](https://github.com/edx-ops/business-secure). I guess if we are going to do that, why store the password in the database at all?

Reply:
yea, we could just store the encrypted password, which would be better. i'll start exploring that route.